team/bundles/org.eclipse.team.ui/src/org/eclipse/team/internal/ui/synchronize/patch/ApplyPatchOperation.java

```diff
@@ -229,8 +229,7 @@ public void run() {
 
 	private static IFilePatch[] internalParsePatch(IStorage storage)
 			throws CoreException {
-		BufferedReader reader = Utilities.createReader(storage);
-		try {
+		try (BufferedReader reader = Utilities.createReader(storage)) {
```

Comment:
Same as above: This is not behavior-preserving. Before, an `IOException` during `close()` did nothing, now it may throw a `CoreException`.